frontend_app_admin_workspace_user/src/css/index.styl

```diff
@@ -19,9 +19,18 @@
     &:hover, &:nth-child(even):hover
       background-color grey-hover
     &__td-link
+      max-width 120px
+      white-space nowrap
+      overflow hidden
+      text-overflow ellipsis
       cursor pointer
       &:not(:hover)
         color inherit
+    &__td-description
+      max-width 440px
```

Comment:
This value doesn't work on small devices.
You should add some breakpoint to reduce it.